core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -152,4 +172,129 @@ public Response getStartTimeMs() {
   public Response getVersion() {
     return Response.ok(Version.VERSION).build();
   }
+
+  /**
+   * @summary get the total capacity of all workers in bytes
+   * @return the response object
+   */
+  @GET
+  @Path(GET_CAPACITY_BYTES)
+  @ReturnType("java.lang.Long")
+  public Response getCapacityBytes() {
+    return Response.ok(mBlockMaster.getCapacityBytes()).build();
+  }
+
+  /**
+   * @summary get the total disk capacity in bytes, a negative value means the capacity is unknown.
+   * @return the response object
+   */
+  @GET
+  @Path(GET_DISK_CAPACITY_BYTES)
+  @ReturnType("java.lang.Long")
+  public Response getDiskCapacityBytes() {
+    try {
+      return Response.ok(mUfs.getSpace(mUfsRoot, UnderFileSystem.SpaceType.SPACE_TOTAL)).build();
+    } catch (IOException e) {
+      LOG.warn(e.getMessage());
+      return Response.serverError().entity(e.getMessage()).build();
+    }
+  }
+
+  /**
+   * @summary get the free disk capacity in bytes, a negative value means the capacity is unknown.
+   * @return the response object
+   */
+  @GET
+  @Path(GET_DISK_FREE_BYTES)
+  @ReturnType("java.lang.Long")
+  public Response getFreeDiskCapacityBytes() {
+    try {
+      return Response.ok(mUfs.getSpace(mUfsRoot, UnderFileSystem.SpaceType.SPACE_FREE)).build();
+    } catch (IOException e) {
+      LOG.warn(e.getMessage());
+      return Response.serverError().entity(e.getMessage()).build();
+    }
+  }
+
+  /**
+   * @summary get the used disk capacity, a negative value means the capacity is unknown.
+   * @return the response object
+   */
+  @GET
+  @Path(GET_DISK_USED_BYTES)
+  @ReturnType("java.lang.Long")
+  public Response getUsedDiskCapacityBytes() {
+    try {
+      return Response.ok(mUfs.getSpace(mUfsRoot, UnderFileSystem.SpaceType.SPACE_USED)).build();
+    } catch (IOException e) {
+      LOG.warn(e.getMessage());
+      return Response.serverError().entity(e.getMessage()).build();
+    }
+  }
+
+  /**
+   * @summary get the free capacity
+   * @return the response object
+   */
+  @GET
+  @Path(GET_FREE_BYTES)
+  @ReturnType("java.lang.Long")
+  public Response getFreeCapacityBytes() {
+    return Response.ok(mBlockMaster.getCapacityBytes() - mBlockMaster.getUsedBytes()).build();
+  }
+
+  /**
+   * @summary get the mapping from tier alias to total capacity of the tier in bytes
+   * @return the response object
+   */
+  @GET
+  @Path(GET_CAPACITY_BYTES_ON_TIERS)
+  @ReturnType("java.util.Map<String, Long>")
+  public Response getCapacityBytesOnTiers() {
```

Comment:
The methods are not necessarily sorted alphabetically, similar methods like methods for getting capacities are put together.